docs/enterprise/getstarted/migrate-from-oss.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Migrate from lakeFS OSS
+description: How to migrate from lakeFS OSS to lakeFS Enterprise
+parent: Get Started
+grand_parent: lakeFS Enterprise
+nav_order: 203
+---
+
+# Migrate From lakeFS Open-Source to lakeFS Enterprise
+
+To migrate from lakeFS Open Source to lakeFS Enterprise, follow the steps below:
+
+1. [Contact us](https://lakefs.io/contact-sales/) to gain access to Fluffy. You will be granted with a token that enables downloading *dockerhub/fluffy* from [Docker Hub](https://hub.docker.com/u/treeverse).
+1. Sanity Test (Optional): Install a fresh lakeFS enterprise with a fresh database. Test the setup > login > Create repository etc. Once everything seems to work, delete and cleanup the test setup and we will move to the migration process.
+1. Follow lakeFS [Enterprise installation guide][lakefs-enterprise-install]
+   1. Make sure that you meet the [prerequisites][lakefs-enterprise-install-prerequisites]
+   1. Create a customized values.yaml file for your deployment
+   1. Hold before [installing the lakeFS Helm chart][lakefs-enterprise-install-helm-install]
```

Comment:
This line is irrelevant saying "don't do an action" and wondering "when then?" -  IMO it's OK to omit this line.